include/gzip/decompress.hpp

```diff
@@ -83,15 +87,19 @@ class Decompressor {
         inflateEnd(&inflate_s);
         output.resize(size_uncompressed);
     }
-
 };
 
-std::string decompress(const char * data, std::size_t size)
+inline std::string decompress(const char* data, std::size_t size)
 {
     Decompressor decomp;
     std::string output;
-    decomp.decompress(output,data,size);
+    decomp.decompress(output, data, size);
     return output;
 }
 
+inline std::string decompress(std::string const& input)
+{
+    return decompress(input.data(), input.size());
+}
```

Comment:
@flippmoke going to revert this addition since it was removed intentionally per https://github.com/mapbox/gzip-hpp/pull/1#issuecomment-326078576 /cc @GretaCB

Reply:
done in a217d04